python/src/aiconfig/editor/client/src/themes/LocalTheme.ts

```diff
@@ -0,0 +1,167 @@
+import { MantineThemeOverride } from "@mantine/core";
```

Comment:
What does override mean here? I tried looking at the docs but wasn't sure what exactly it would be "overriding"

Reply:
Mantine has a default theme if no MantineProvider is used to override it; the default has primary color of blue for buttons, hover outlines, etc., as well as specific fonts for text